plugins/catalog-backend/scripts/mock-data.sh

```diff
@@ -17,3 +17,11 @@ for URL in \
     --data-raw "{\"type\": \"github\", \"target\": \"https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/${URL}\"}"
   echo
 done
+
+curl \
+    --location \
+    --request POST 'localhost:7000/catalog/locations' \
+    --header 'Content-Type: application/json' \
+    --data-raw "{\"type\": \"github\", \"target\": \"https://github.com/benjdlambert/cookiecutter-golang/blob/master/template.yaml\"}"
```

Comment:
Need to press foss internally so we can move this to spotify repo rather than mine.

Reply:
Not sure if this will block us merging this or not? What do you think @stefanalund + @spotify/backstage-core?